source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -0,0 +1,117 @@
+# Mattermost Cloud Changelog
+
+This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
+
+## Release 2020-12-09
+
+### Highlights
+
+#### Incident Management provided out-of-the-box (E20)
+ - Pre-packaged and pre-installed the Incident Management as well as Channel Export plugins for enterprise-ready builds.
+
+### Improvements
+
+#### Cloud-specific
+ - The Customer Web Server can now make HTTP requests to an installation by passing the X-Cloud-Token header, with the value being that installation's Customer API Key.
+ - System Admins can receive an email notifying them of a failed payment in Mattermost cloud.
+ - In the event a subscription has a payment and there is no card on the account when that payment is processed, an email will be sent to attempt to get the user to pay up.
+
+#### User Interface (UI)
+ - Added the ability to mute categories with the experimental sidebar feature.
+ - Added support for multi-selection of channels for dragging and dropping between channels in the experimental sidebar feature.
+ - Group messages are now returned in the channel switcher when only first names are typed.
+ - /dnd command will only set a do not disturb status whatever the current user status is.
+
+#### Administration
+ - Added a new `manage_remote_clusters` permission.
+ - Enabled goSAML2 library as the only supported SAML Library.
+
+### Bug Fixes
+ - Cleaned up the config store on server initialization errors.
+ - Fixed an issue where permissions did not grant read and/or write access to the Global Relay configuration settings.
+ - Fixed an issue where the site configuration ‘’Read only’’ permission did not make the "Notice" section as read-only for the System Manager.
+ - Fixed an issue where importing Client4 in a node server caused an exception due to rudder modules.
+ - Fixed an issue where LDAP ‘’FirstLoginSync’’ didn't close LDAP Session.
+
+### Database Changes
+ - Added ``UnreadMentions`` column to ``ThreadMembership`` table.
+ 
+### Websocket Event Changes
+ - New websocket events added: ``thread_updated``, ``thread_follow_changed``, ``thread_read_changed``.
```

Comment:
@jwilander Interested in your input, but these are not likely necessary for customer-facing, since they don't have access to the database or likely care about Websocket events.

Reply:
Correct, this is getting pretty technical and won't inform admins/end users at all